File: Programing_Basics_with_JS/conditionalStatementsAdvancedMoreExercises/05.Vacation.js
function vacation(input) {
    let budget = Number(input[0]);
    let season = input[1];

    let placeToStay = '';
    let location = '';
    let vacationPrice = 0;

    if (budget <= 1000) {
        placeToStay = 'Camp';
        if (season === 'Summer') {
            location = 'Alaska';
            vacationPrice = budget * 0.65;
        } else {
            location = 'Morocco';
            vacationPrice = budget * 0.45;
        }
    } else if (budget <= 3000) {
        placeToStay = 'Hut';
        if (season === 'Summer') {
            location = 'Alaska';
            vacationPrice = budget * 0.80;
        } else {
            location = 'Morocco';
            vacationPrice = budget * 0.60;
        }
    } else {
        placeToStay = 'Hotel';
        vacationPrice = budget * 0.90;
        if (season === 'Summer') {
            location = 'Alaska';
        } else {
            location = 'Morocco';
        }
    }

    console.log(`${location} - ${placeToStay} - ${vacationPrice.toFixed(2)}`);
}

vacation(['800', 'Summer'])
vacation(['799.50', 'Winter'])
vacation(['3460', 'Summer'])
vacation(['1100', 'Summer'])
vacation(['5000', 'Winter'])
vacation(['2543.99', 'Winter'])

File: Programming_Basics_with_Python/conditional_statements_advanced_more_exercises/05._vacation.py
budget = float(input())
season = input()

place_to_stay = ''
country_to_go = ''
money_cost = 0

if budget <= 1000:
    place_to_stay = 'Camp'
    if season == 'Summer':
        country_to_go = 'Alaska'
        money_cost = budget * 0.65
    elif season == 'Winter':
        country_to_go = 'Morocco'
        money_cost = budget * 0.45
elif 1000 < budget <= 3000:
    place_to_stay = 'Hut'
    if season == 'Summer':
        country_to_go = 'Alaska'
        money_cost = budget * 0.80
    elif season == 'Winter':
        country_to_go = 'Morocco'
        money_cost = budget * 0.60
elif budget > 3000:
    place_to_stay = 'Hotel'
    money_cost = budget * 0.90
    if season == 'Summer':
        country_to_go = 'Alaska'
    elif season == 'Winter':
        country_to_go = 'Morocco'

print(f'{country_to_go} - {place_to_stay} - {money_cost:.2f}')

